packages/components/src/components/input-date/input-date.e2e.ts

```diff
@@ -60,6 +61,158 @@ test.describe('kol-input-date', () => {
 		});
 	});
 
+	test.describe('Consistent Return Value for type datetime-local', () => {
```

Comment:
Die Beschreibung passt nicht so ganz: Wir testen keine return values (außer bei getValue) und spezifizieren nicht den type datetime-local.
Vorschlag: Ich würde den Block einfach "Events" nennen.
```suggestion
	test.describe('Events', () => {
```